worlds/Arena.wbt

```diff
@@ -26,17 +26,17 @@ DEF AMBIENT Background {
 }
 
 # Updating colours? Update `territory_controller.py`
-DEF ROBOT-0 SRRobot {
-  translation 0 0 -7
-  rotation 0 1 0 3.14
+DEF ROBOT-0 SBForklift {
```

Comment:
Should this PR also delete the SRRobot proto file (and associated textures etc.)?

Reply:
I was keeping SRRobot around for the moment so that modifying the code to support two different robots on each team could be done before the crane robot is created.

Reply:
👍